src/modules/control/src/ardu_adapter.py

```diff
@@ -69,40 +64,51 @@ def high_lvl_callback(high_lvl_control_msg):
 
     global high_lvl_steering_callback
     high_lvl_steering_callback = ((-1)*steering_angle*180/pi+steering_analog_intercept)/steering_analog_slope+potentiometer_offset
-
-'''
-def driving_high_lvl_callback(high_driving_msg):
-    
-    #Desired PWM Value from the high lvl control
     
-    high_lvl_driving_callback = (high_driving_msg.data-min_driving_input)*(max_driving_output-min_driving_output)/(max_driving_input-min_driving_input)+min_driving_output
-def steering_high_lvl_callback(high_steering_msg):
+    '''
+def timer_callback(event):
+    '''Handles the conversion between steering in radians from the high level control and analog value for the low level control
+    '''
+    global high_lvl_driving_callback
+    if driving_msg >= 0:
+        high_lvl_driving_callback = (driving_msg-min_driving_input)*(max_driving_output-min_driving_output)/(max_driving_input-min_driving_input)+min_driving_output
+    else:
+        high_lvl_driving_callback = (-1)*(abs(driving_msg)-min_driving_input)*(max_driving_output-min_driving_output)/(max_driving_input-min_driving_input)+min_driving_output
+    steering_angle = 0
+
+    if(abs(steering_msg) > steering_limit_radians): # establishing limits in radians for steering
+        steering_angle = steering_limit_radians*(abs(steering_msg)/steering_msg)
+    else:
+        steering_angle = steering_msg 
+
+    global high_lvl_steering_callback
+    high_lvl_steering_callback = ((-1)*steering_angle*180/pi+steering_analog_intercept)/steering_analog_slope+potentiometer_offset
     
-    #Converts steering angle required from high lvl control to analog input for the linear actuator
-    high_lvl_steering_callback = (high_steering_msg.data*180/pi + 36.15)/.1721
-'''
+    steering_low_lvl_node_publisher.publish(high_lvl_steering_callback)
+    driving_low_lvl_node_publisher.publish(high_lvl_driving_callback)
 
-def Control_Adapter_Arduino_node():
+def Control_Adapter_Arduino_Node():
```

Comment:
change each first letter to lower case
```suggestion
def control_adapter_arduino_node():
```